.github/workflows/master.yml

```diff
@@ -56,6 +56,7 @@ jobs:
     - name: NPM Publish
       id: publish
       uses: JS-DevTools/npm-publish@v1
+      if: ${{ steps.branch-name.outputs.current_branch == 'master' }}
```

Comment:
@ryankazokas I added this here so it won't automatically publish when we open a PR.  This way it will only publish when we merge.

Let me know if that's ok.  If not I can remove.

Reply:
i think we should still try to publish develop branches if we can. npm link works for the webapp but playbook is a bit harder to test locally, so it would be better to test in a staging environment before we do a larger scale deploy.

Reply:
If you're going to publish a dev branch then the [tag](https://github.com/ChannelApe/channelape-typescript-sdk/blob/master/.github/workflows/master.yml#L56-L61) at the very least shouldn't be latest then. It should be latest if it's master and develop otherwise.